### PR TITLE
Add missing binaries in TRANSLATIONS.md instructions.

### PR DESCRIPTION
**Pull request by yohcop**
With these instructions, I can start from a clean fork, and get running quickly.

### CODE REVIEW

**Comment by henrif75** [comment]
I'm just realizing that the whole "Preparation" section is redundant since it's already on the README file. Perhaps you could add the Ubuntu info to the README and replace the "Preparation", for a note asking people to ensure they've followed instructions there?

File: TRANSLATIONS.md [hunk 1]
```diff
@@ -27,17 +27,24 @@ GNU Gettext utilities below.
 
 ## Preparation
 
-We use two helpers for the translations:
+You will need the [Gettext] utilities (`msginit`, `msgmerge`). Under Ubuntu, you
```
Comment (mgeisler) [inline TRANSLATIONS.md:30]: We use a Debian-based distribution at work, so I would like us to mention Debian here:

```suggestion
You will need the [Gettext] utilities (`msginit`, `msgmerge`). Under Debian and Ubuntu, you
```

**Comment by mgeisler** [comment]
> I'm just realizing that the whole "Preparation" section is redundant since it's already on the README file. Perhaps you could add the Ubuntu info to the README and replace the "Preparation", for a note asking people to ensure they've followed instructions there?

I tried to cover two use-cases here: README is for people who would like to read the course offline. We've managed to make the translation infrastructure mostly orthogonal to that usecase, so the README doens't need to talk about the Gettext tools. It _almost_ doesn't have to talk about mdbook-i18n-helpers: things work if you don't install them, but you get a warning from `mdbook build`. So that's why mdbook-i18n-helpers is added to the README (I would like to remove the mdbook-exerciser, see #625).

I see TRANSLATIONS as the main document for people who want to contribute to the translations. So it needs to mention Gettext and the rest of the infrastructure. As for installing mdbook, we could refer back to the README here. That would be better since we then keep the instructions in one place.

How does that sound?

**Comment by yohcop** [comment]
Yes, I think I understand it the same way as @mgeisler . TRANSLATIONS.md is instructions for translators, README for users/contributors to the course.

File: TRANSLATIONS.md [hunk 2]
```diff
@@ -27,17 +27,24 @@ GNU Gettext utilities below.
...
   renderer.
 - `mdbook-gettext`: This program translates the book into a target language. It
   is an mdbook preprocessor.
 
-Install both helpers with:
+Install everything with:
 
 ```shell
-$ cargo install mdbook-i18n-helpers
+$ cargo install mdbook mdbook-i18n-helpers
```
Comment (mgeisler) [inline TRANSLATIONS.md:47]: I think we should leave this as it is and instead refer back to the README for how to build the book (as @henrif75 suggested). So the instructions here could start with something like:

> First, ensure you can build the book and that `mdbook serve` works (follow the instructions in the README).

Then everything will be nicely split.

Comment (yohcop) [reply TRANSLATIONS.md]: Ok, let me know if that works!

**Review by henrif75 — APPROVED** [review]

**Review by mgeisler — APPROVED** [review]

**Comment by mgeisler** [comment]
Thanks!

**Pull request merged**